intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/plugin/IdeMode.kt

```diff
@@ -0,0 +1,28 @@
+package com.jetbrains.plugin.structure.intellij.plugin
+
+sealed class IdeMode(val isK1Compatible: Boolean, val isK2Compatible: Boolean) {
```

Comment:
rename to `KotlinPluginMode` or similar?

Reply:
Renamed to `KotlinPluginMode`.